docs/presets/community.md

```diff
@@ -1,54 +0,0 @@
-# Community presets
```

Comment:
I think maybe we could keep this page, but replace the content to link the community